content/english/chatbot/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Create your own chatbot"

```

Comment:
Please also include an answer key for this workshop

Reply:
don't have time to do this before tomorrow, but can add after